cdap-ai-ext-vertexai/src/test/java/io/cdap/cdap/ai/ext/vertexai/VertexAIConfigurationTest.java

```diff
@@ -0,0 +1,153 @@
+package io.cdap.cdap.ai.ext.vertexai;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for the {@link VertexAIConfiguration} class.
+ *
+ * This class contains tests to verify the behavior of the {@link VertexAIConfiguration} class
+ * when initialized with various sets of properties.
```

Comment:
No need to add details about the tests. Otherwise, this comment will need an update when new tests are added.

Reply:
Done